docs/CHANGELOG.md

```diff
@@ -9,6 +9,10 @@ title: Changelog
 
 ## main
 
+* Add support for Capybara finders and other helpers
```

Comment:
Might we include examples of the finders and helpers that we're adding support for?

Reply:
Sure, I'll try and do that.

Reply:
Added: 4a859874.